<?xml version="1.0"?>
<launch>
    <arg name ="rosbag_file" default="/mnt/c/wslShared/Robotics/lidar/lidar-imu-slam/env_ws/bag/rad_test_1.bag"/>
    <arg name = "config_loc" default = "$(find limu)/config/params.yaml"/>
    <arg name="launch_rviz" default="false"/>

    <!--Launch module-->
    <node pkg="limu" type="lio" name="lio" output="screen">
        <param name="config_loc" value= "$(arg config_loc)"/>
    </node>

    <!--Launch rosbag-->
    <node pkg="rosbag" type="play" name="player" output="screen" args="--clock $(arg rosbag_file)"/>

    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find limu)/rviz/rviz.rviz" if="$(arg launch_rviz)"/>

</launch>